client-src/elements/chromedash-feature-table.ts

```diff
@@ -1,52 +1,49 @@
 import {LitElement, css, html, nothing} from 'lit';
-import {showToastMessage, clamp} from './utils.js';
+import {customElement, property, state} from 'lit/decorators.js';
+import {SHARED_STYLES} from '../css/shared-css.js';
+import {Feature} from '../js-src/cs-client.js';
 import './chromedash-feature-filter';
 import './chromedash-feature-row';
-import {SHARED_STYLES} from '../css/shared-css.js';
+import {clamp, showToastMessage} from './utils.js';
 
+@customElement('chromedash-feature-table')
 class ChromedashFeatureTable extends LitElement {
-  static get properties() {
-    return {
-      query: {type: String},
-      showEnterprise: {type: Boolean},
-      sortSpec: {type: String},
-      showQuery: {type: Boolean},
-      features: {type: Array},
-      totalCount: {type: Number},
-      loading: {type: Boolean},
-      reloading: {type: Boolean},
-      start: {type: Number},
-      num: {type: Number},
-      alwaysOfferPagination: {type: Boolean},
-      columns: {type: String},
-      signedIn: {type: Boolean},
-      canEdit: {type: Boolean},
-      starredFeatures: {type: Object},
-      noResultsMessage: {type: String},
-      gates: {type: Object},
-      selectedGateId: {type: Number},
-    };
-  }
-
-  constructor() {
-    super();
-    this.query = '';
-    this.showEnterprise = false;
-    this.sortSpec = '';
-    this.showQuery = false;
-    this.loading = true;
-    this.reloading = false;
-    this.starredFeatures = new Set();
-    this.features = [];
-    this.totalCount = 0;
-    this.start = 0;
-    this.num = 100;
-    this.alwaysOfferPagination = false;
-    this.noResultsMessage = 'No results';
-    this.canEdit = false;
-    this.gates = {};
-    this.selectedGateId = 0;
-  }
+  @state()
+  query = '';
```

Comment:
This can't be `@state` because it's set as an attribute at https://github.com/GoogleChrome/chromium-dashboard/blob/6e446f44ce481954865849d14c6f5cf06b222216/client-src/elements/chromedash-myfeatures-page.js#L86-L87

Please double-check the [3 uses of this element](https://github.com/search?q=repo%3AGoogleChrome%2Fchromium-dashboard%20%22%3Cchromedash-feature-table%22&type=code) to make sure none of them are setting fields that you've marked as `@state`. If they're setting it as a `.property` (especially objects and arrays that can't be safely passed as attributes), you can use `@property({attribute:false})`.